src/auth/auth.service.ts

```diff
@@ -5,16 +5,16 @@ import {
   Injectable,
   UnauthorizedException,
 } from '@nestjs/common';
-import { PrismaService } from '../prisma/prisma.service';
+import { ConfigService } from '@nestjs/config';
 import { JwtService, TokenExpiredError } from '@nestjs/jwt';
-import { EmailUpdateDto, LoginDto, RegisterDto } from './dto';
-import { Response } from 'express';
-import * as bcrypt from 'bcryptjs';
-import { VerificationSerivce } from './verification.service';
 import { User } from '@prisma/client';
-import { GlobalException } from 'src/exceptions/global.exception';
+import * as bcrypt from 'bcryptjs';
+import { Response } from 'express';
 import ErrorsTypes from 'src/errors/errors.enum';
-import { ConfigService } from '@nestjs/config';
+import { GlobalException } from 'src/exceptions/global.exception';
+import { PrismaService } from '../prisma/prisma.service';
+import { EmailUpdateDto, LoginDto, RegisterDto } from './dto';
+import { VerificationSerivce } from './verification.service';
```

Comment:
please order imports to have service near to service and so on. little change but saves time of development